data/json/items/obsolete.json

```diff
@@ -2755,5 +2755,29 @@
     "bashing": 8,
     "to_hit": -4,
     "category": "veh_parts"
+  },
+  {
+    "type": "GENERIC",
+    "id": "rag_bloody",
+    "symbol": ",",
+    "color": "red",
+    "name": { "str": "blood soaked rag" },
+    "description": "A large rag, drenched in blood.  It could be cleaned with boiling water.",
+    "material": [ "cotton" ],
+    "flags": [ "NO_SALVAGE", "TRADER_AVOID" ],
+    "weight": "80 g",
+    "volume": "250 ml"
+  },
+  {
```

Comment:
obsolete recipe should move to there https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/recipes/recipe_obsolete.json